data/projects/bdk.mdx

```diff
@@ -14,4 +14,4 @@ tags: ['Bitcoin', 'Library']
 
 ## About this project
 
-Bitcoin Development Kit (BDK) is a set of libraries and tools that allows you to seamlessly build cross-platform on-chain Bitcoin wallets without having to re-implement standard bitcoin data structures, algorithms, and protocols. We partner closely with the Lightning Dev Kit (LDK) team to help projects building on-chain and off-chain wallets. BDK is built on top of the powerful rust-bitcoin and rust-miniscript libraries and adds features for managing descriptor based wallets, syncing wallets to the bitcoin blockchain, viewing transaction histories, managing and selecting utxos to create new transactions, signing, and more. The core BDK components are written in Rust, but the team also maintains Kotlin and Swift language binding for use in mobile projects. Plus we maintain Python bindings and React Native and Flutter support is being developed. 😃 Join our community Open source is fundamental to this project and we would love to connect with you. Whether you're a Rust, Kotlin, Swift, or Python developer, or good at technical writing, web design or want to help via manual testing. Everyone is welcome. Most of our communication happens on the Bitcoin Dev Kit discord server, say hi!
```

Comment:
Small change of adding "!" after the word community. There was no punctuation there. 

"😃 Join our community! Open source is fundamental to this project and we would love to connect with you. Whether you're a Rust, Kotlin, Swift, or Python developer, or good at technical writing, web design or want to help via manual testing."

Reply:
Again, I'm using this opportunity to see if I am understanding the workflow in GitHub. The sections in "red" are the ones being removed from the code, so no need to make any edits/commits to this section, unless you would like to put it (in full or in part) back into the code, in which case you would have to make your commit to my PR. I hope I'm making sense...

Reply:
What Lucas said. 

Red = removed
Green = added

To quote [the docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-comparing-branches-in-pull-requests#three-dot-and-two-dot-git-diff-comparisons):

> Additions to the files appear in green and are prefaced by a `+` sign while content that has been removed appears in red and is prefaced by a `-` sign.

Reply:
What Lucas said. 

Red = removed
Green = added

To quote [the docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-comparing-branches-in-pull-requests#three-dot-and-two-dot-git-diff-comparisons):

> Additions to the files appear in green and are prefaced by a `+` sign while content that has been removed appears in red and is prefaced by a `-` sign.

Reply:
What Lucas said. 

Red = removed
Green = added

To quote [the docs](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-comparing-branches-in-pull-requests#three-dot-and-two-dot-git-diff-comparisons):

> Additions to the files appear in green and are prefaced by a `+` sign while content that has been removed appears in red and is prefaced by a `-` sign.